src/plugins/dataflow/model/dataflow-content.ts

```diff
@@ -47,7 +47,9 @@ export const DataflowContentModel = TileContentModel
     type: types.optional(types.literal(kDataflowTileType), kDataflowTileType),
     program: types.optional(DataflowProgramModel, getSnapshot(DataflowProgramModel.create())),
     programDataRate: DEFAULT_DATA_RATE,
+    programRecordingMode: 0,
```

Comment:
I thought we decided not to store this in the state of the model and just use react state for it.